changelog.md

```diff
@@ -15,6 +15,7 @@
 * `FIX` Fixed the error that the configuration file pointed to by the `--configpath` option was not read and loaded.
 * `FIX` Generic return can be optional.
 * `FIX` Fixed the comment calculating in docs `---@param a string?Comment` - now its `Comment` instead of `omment`.
+* `FIX` FIxed cannot bind varibles using tail comment `@class` [#2673](https://github.com/LuaLS/lua-language-server/issues/2673)
```

Comment:
Fixed

Reply:
good catch, seems I didn't toggle the caplock correctly when I type `Fixed` 😂 

I just proofread this sentence again, and found that there is **another typo** `varibles` => `variables`🤦‍♂️ 
I will amend it together